docs/guides/advanced/fetcher.md

```diff
@@ -301,6 +316,18 @@ try {
 }
 ```
 
+
+### Headers
+
+The HTTP headers returned can be accessed using the `headers` field of the response. The header names will all be lowercase, regardless of how they are sent by the server.
```

Comment:
I might describe as being "normalized for convenience".

I believe that the request-promise library is what does this, and my understanding is it's meant to be a feature rather than a quirk, so you don't have to guess at the casing.

Reply:
Good point, done.